### PR TITLE
Merges gitbook documentation

### LINKED ISSUE #13: Data from Gitbook is not migrated
Self explanatory. Data from Gitbook has not yet been migrated to these docs.

### PR DESCRIPTION
**Pull request by Exerra**
This PR will merge gitbook documentation with these docs. Currently a few simple things are added but I will need to spruce it up

### CODE REVIEW

**Comment by Exerra** [comment]
The main content is here. Now I will need to migrate the new settings JSON to my API and, well, spruce the new sections up a bit.

**Review by Amiichuu — COMMENTED** [review]
So far so good, I suggest sprucing it up a bit. Maybe display the swear words and crypto in a table?

**Comment by Exerra** [comment]
> So far so good, I suggest sprucing it up a bit. Maybe display the swear words and crypto in a table?

Thought about it, but ngl unless there are going to be severity options, having a one column table is weird

File: users/supportedCrypto.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Supported crypto currencies
+
+> Didn't know Karen rejected fiat currencies too
```
Comment (Amiichuu) [inline users/supportedCrypto.md]: Change this to "Karen Bot"

File: users/supportedCrypto.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* Bitcoin
+* Ethereum
+* Litecoin
+* Dogecoin
+* Dash
+* Ripple
+* Groestlcoin
+* Stellar
+
+... and their short versions (e.g btc, eth and so on)!
```
Comment (Amiichuu) [inline users/supportedCrypto.md]: Capitalize the short versions

File: users/swearWords.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Swear words
+
+> Karen filters swear words? Oh the irony
```
Comment (Amiichuu) [inline users/swearWords.md]: Again, branding

File: users/swearWords.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+So this section focuses on the swear words Karen Bot filters if you have "AntiSwear" enabled in the settings (by default it is off). All of these messages result in a deletion of the message and a... *pleasant* message sent in that channel.
```
Comment (Amiichuu) [inline users/swearWords.md]: Link "settings" to `users/settings.md`

**Review by Amiichuu — CHANGES_REQUESTED** [review]
Please change the things I provided and re-request review when you're done

File: users/supportedCrypto.md (hunk 1 above)
Comment (Exerra) [reply users/supportedCrypto.md]: Forgot about that
Done

File: users/supportedCrypto.md (hunk 2 above)
Comment (Exerra) [reply users/supportedCrypto.md]: Done

**Review by Amiichuu — APPROVED** [review]
Issues are now fixed, approved

File: users/swearWords.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Swear words
+
+> Karen Bot filters swear words? Oh the irony
+
+So this section focuses on the swear words Karen Bot filters if you have "AntiSwear" enabled in the [settings](users/settings.md) (by default it is off). All of these messages result in a deletion of the message and a... *pleasant* message sent in that channel.
```
[the remark this answers is not in the document]
Comment (Amiichuu) [reply users/swearWords.md:5]: It works with docsify, so no

**Pull request merged**